lib/main.js

```diff
@@ -88,10 +88,19 @@ define(['moment', 'utils/router', 'leaflet', 'gui', 'helper', 'utils/language'],
 
       update()
         .then(function (d) {
-          var gui = new GUI(language);
-          gui.setData(d);
-          router.setData(d);
-          router.resolve();
+          var gui;
+
+          function ensureLanguageLoaded() {
+            if (_.phrases.yes === undefined) {
+              window.setTimeout(ensureLanguageLoaded, 20);
+            }
+            gui = new GUI(language);
+            gui.setData(d);
+            router.setData(d);
+            router.resolve();
+          }
+
+          ensureLanguageLoaded();
```

Comment:
Falls die Sprachr nicht geladen werden kann, crashed allles nachdem window setInterval die meshviewer.json geholt hat.

Reply:
Was willst du sonst machen

Reply:
mit in `ensureLanguageLoaded` nehmen

Reply:
zudem in Zeile 96 ein return einbauen -> damit er die gui nicht immer neu baut

Reply:
was willst du mit reinnehmen? den init der language?
Laedt nur einmal, der reload kommt spaeter

Reply:
Der der window.setInterval aufruf.